stock-price-checker/README.md

```diff
@@ -1,10 +1,30 @@
-**FreeCodeCamp**- Information Security and Quality Assurance
-------
+# Stock Price Checker
 
-Project Stock Price Checker
+## User stories:
 
-1) SET NODE_ENV to `test` without quotes and set DB to your monog connection string
-3) You will add any security features to `server.js`
-4) You will create all of the functional/unit tests in `tests/2_functional-tests.js` and `tests/1_unit-tests.js`
+1. Set the content security policies to only allow loading of scripts and CSS from your server.
+1. I can `GET` `/api/stock-prices` with form data containing a Nasdaq `stock` ticker and receive back an object `stockData`.
+1. In `stockData`, I can see the `stock` (the ticker as a string), `price` (decimal in string format), and `likes` (int).
+1. I can also pass along the field `like` as `true` (boolean) to have my like added to the stock(s). Only 1 like per IP should be accepted.
+1. If I pass along 2 stocks, the return object will be an array with information about both stocks. Instead of `likes`, it will display `rel_likes` (the difference between the likes) on both.
+1. A good way to receive current prices is through our stock price proxy (replacing 'GOOG' with your stock symbol): `https://stock-price-checker-proxy.freecodecamp.rocks/v1/stock/GOOG/quote`
+1. All 5 functional tests are complete and passing.
 
+## Example usage:
+
+- `/api/stock-prices?stock=GOOG`
+- `/api/stock-prices?stock=GOOG&amp;like=true`
+- `/api/stock-prices?stock=GOOG&amp;stock=MSFT`
+- `/api/stock-prices?stock=GOOG&amp;stock=MSFT&amp;like=true`
+
+## Example return:
+
+- `{ "stockData": { "stock": "GOOG", "price": "786.90", "likes": 1 } }`
+- `{ "stockData": [{"stock": "MSFT", "price": "62.30", "rel_likes": -1 }, { "stock": "GOOG", "price": "786.90", "rel_likes": 1 }] }`
+
+## Additional notes:
+
+- SET NODE_ENV to `test` without quotes and set DB to your Mongo connection string
+- You will add any security features to `server.js`
+- You will create all of the functional/unit tests in `tests/2_functional-tests.js` and `tests/1_unit-tests.js`
```

Comment:
Wondering if we should have stuff like this in the readme or on the webpage? This PR would have them in both place on this project and the others with similar example usages. Perhaps we should just have them in one or the other.

Reply:
@moT01, thanks for pointing this out. I spoke briefly with @RandellDawson and we decided to keep the examples on the landing page to reduce duplication. The proxies definitely need examples on the page, so having everything there will help keep things consistent. I'll go ahead and remove the example usage and return sections from the README files.

@RandellDawson mentioned that it might be possible to move the additional notes section to the project pages on /learn, and update them automatically like the `testText`/user stories down the road. I'll leave them in the README for now if that's alright, but just let me know and I can start moving them somewhere else.